fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/ApplicationAmbassador.java

```diff
@@ -678,7 +682,6 @@ private void addVehicleIfNotYetAdded(long time, String unitName) {
         final VehicleRegistration vehicleRegistration = vehicleRegistrations.remove(unitName);
         if (vehicleRegistration != null) {
             UnitSimulator.UnitSimulator.registerVehicle(time, vehicleRegistration);
-            SimulationKernel.SimulationKernel.getCentralPerceptionComponent().addVehicle(vehicleRegistration);
```

Comment:
Why can we remove this?

Reply:
This method was only required to fetch the vehicle type from the vehicle registration before it is updated and moved.
Now the type information is cached as soon as the vehicle registration is received in `process(final VehicleRegistration vehicleRegistration)`, which removes the necessity of the `addVehicle`-method